thirdeye/thirdeye-frontend/app/pods/manage/alerts/edit/controller.js

```diff
@@ -245,7 +254,10 @@ export default Ember.Controller.extend({
     'isDuplicateEmail',
     'isProcessingForm',
     function() {
-      return this.get('isEmptyEmail') || this.get('isProcessingForm') || this.get('isEmailError' || this.get('isDuplicateEmail'));
```

Comment:
fyi: this should be shorter to write: 
```
isSubmitDisabled: Ember.computed.or('{isEmptyEmail, isEmailError, isDuplicateEmail ,isProcessingForm}'),
```
https://www.emberjs.com/api/ember/2.15/namespaces/Ember.computed/methods/bool?anchor=or

Reply:
thanks